telemetry/src/main/java/com/newrelic/telemetry/TelemetryClient.java

```diff
@@ -36,14 +36,16 @@
 public class TelemetryClient {
 
   private static final Logger LOG = LoggerFactory.getLogger(TelemetryClient.class);
+  private static final long DEFAULT_SHUTDOWN_SECONDS = 3L;
```

Comment:
Since this is seconds, let's make this an integer instead of a long.
(Sorry I didn't catch this the first time around)

Reply:
fixed this 🙇 
the reason I used long is awaitTermination's signature seems to require long for the length of any time unit. `awaitTermination(long timeout, TimeUnit unit)`

https://docs.oracle.com/javase/9/docs/api/java/util/concurrent/ExecutorService.html#awaitTermination-long-java.util.concurrent.TimeUnit-